src/bundles/peer-locations.js

```diff
@@ -82,16 +82,41 @@ export default function (opts) {
     })
   )
 
-  bundle.selectPeerCoordinates = createSelector(
+  const COORDINATES_RADIUS = 4
```

Comment:
I think I finally understand why it looked odd to me :sweat_smile: 
Radius was much bigger than the biggest circle, so there was too much whitespace between dots in east coast of US, Europe and China.  I expect those areas to be _denser_, so whitespace between dots should be minimal.

(it may be subjective, so feel free to ignore below suggestion)

```suggestion
  const COORDINATES_RADIUS = 2
```

Reply:
Intersecting circles (like it's doing in North America) leads to weird hovering glitches, that's why I kept the minimum safe distance of 4. And that can still overlap if a region has more than 100 peers 😕

Reply:
@lidel I might be inclined to leave as is ... @rafaelramalho19 did a lot of intermediate iterations on this that I got to see and this is kind of the best fit between circle size accuracy and avoiding weird hovering problems. Just my $.02 though 😊

Reply:
Leaving 4 SGTM, especially if it was already considered :-)